### ISSUE TITLE
Column used together with Section add extra td element, using local version works fine

### ISSUE THREAD

**Issue by Philip2809** [resendlabs/react-email#385]
Hi! The Column component adds an extra td element.

```
        <Container style={container}>
          <Section>
            <Column>
              <Section>
                <Img
                  src={`https://birdview.se/logo_b_np.png`}
                  width="120"
                  height="auto"
                  alt="Bird View UF"
                />
              </Section>
              <Section>
                <Text>Bird View</Text>
              </Section>
            </Column>
          </Section>
```

Using this code gives me:
![image](https://user-images.githubusercontent.com/51290462/214775765-dac0d6f7-b842-4565-94ab-3c35898786d7.png)

It seams like for every `<Column>` I add, I get an extra td
```
          <Section>
            <Column>A</Column>
            <Column>B</Column>
            <Column>C</Column>
          </Section>
```
![image](https://user-images.githubusercontent.com/51290462/214775845-7f497379-f547-44bb-983a-a22a235c1cd4.png)

**Comment by Philip2809** [comment]
Rendering out an empty column gives:
![image](https://user-images.githubusercontent.com/51290462/214822394-60b541b0-8bd7-4bfa-a5a8-0571d484515b.png)
which is correct, it seams to become bad when I wrap the column in Section tags it seams to break?
![image](https://user-images.githubusercontent.com/51290462/214823341-2e8b2f1e-193a-472a-a833-34cd9c30a198.png)
It makes like a dubbletd? The code:
![image](https://user-images.githubusercontent.com/51290462/214823406-34f3cc8c-7356-4f27-8990-4f8d9e07999e.png)

**Comment by Philip2809** [comment]
Using a local copy of the section element it works just fine (taken from the github repo)
![image](https://user-images.githubusercontent.com/51290462/214847850-50071d4d-7673-4bbd-a804-3752f31e7098.png)
![image](https://user-images.githubusercontent.com/51290462/214847885-059f883f-3488-4ec5-81ce-db4706bbab5c.png)
But using the copy from the package gives:
![image](https://user-images.githubusercontent.com/51290462/214847945-d47abc6c-e1eb-4f85-9d2e-67744fbeab08.png)

**Comment by Philip2809** [comment]
Any idea what could cause this?

**Comment by pdolezal0** [comment]
> Any idea what could cause this?

Hey @Philip2809, update to latest version of Section component :). https://react.email/docs/changelog#january-24-2023

**Comment by Philip2809** [comment]
Hi! I can't, it says 0.0.4 is the latest. 
![image](https://user-images.githubusercontent.com/51290462/215086942-9e648e07-d743-406d-ab52-48e3d958375e.png)
It also says so on this website:
https://socket.dev/npm/package/@react-email/section

But at least the bug is fixed and I can use a local version until I can download the latest later, not that urgent.

**Comment by webbower** [comment]
Hi. I'm running into the same issue. It looks like this was fixed in v0.0.4 of `Section` but then was undone in v0.0.7 in #456  where the bug was re-introduced.

**Comment by bukinoshita** [comment]
> Hi. I'm running into the same issue. It looks like this was thought to be fixed in v0.0.4 of `Section` but then was undone in v0.0.7 in #456. In v0.0.6, EACH `Column` is being wrapped in an extra `td` and in v0.0.8 it's back to wrapping all `td`s in a single extra `td`.
> 
> ```tsx
> // React code
> <Section>
>     <Column>
>         <Img src={...} height={40} width={150} />
>     </Column>
>     <Column>
>         <Img
>             src={...} height={40} width={150} />
>     </Column>
> </Section>
> ```
> 
>  ```
> 
> <table style="width:100%" align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation">
>     <tbody>
>         <tr style="display:grid;grid-auto-columns:minmax(0, 1fr);grid-auto-flow:column">
>             <td>
>                 <td>
>                     <img src="..." width="150" height="40" style="display:block;outline:none;border:none;text-decoration:none" />
>                 </td>
>             </td>
>             <td>
>                 <td>
>                     <img src="..." width="150" height="40" style="display:block;outline:none;border:none;text-decoration:none" />
>                 </td>
>             </td>
>         </tr>
>     </tbody>
> </table>
> ```
> 
>  ```
> 
> <table align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation" width="100%">
>     <tbody>
>         <tr>
>             <td>
>                 <td>
>                     <img src="..." width="150" height="40" style="display:block;outline:none;border:none;text-decoration:none" />
>                 </td>
>                 <td>
>                     <img src="..." width="150" height="40" style="display:block;outline:none;border:none;text-decoration:none" />
>                 </td>
>             </td>
>         </tr>
>     </tbody>
> </table>
> ```

Thanks for reporting this detailed between version. Latest version is more stable now, can you double check that? Also `Column` should always be used inside of a `Row`.

eg

```tsx
<Section>
  <Row>
    <Column>
      <Img src={...} height={40} width={150} />
    </Column>
    <Column>
      <Img src={...} height={40} width={150} />
    </Column>
  </Row>
</Section>
```

Going to close this issue since latest version fixes this and let me know if any issue pops up and we can reopen
